Documentation/ceph-nfs-crd.md

```diff
@@ -4,19 +4,22 @@ weight: 3100
 indent: true
 ---
 
-# Ceph NFS Gateway CRD
+# Ceph NFS Server CRD
 
 ## Overview
+Rook allows exporting NFS shares of the filesystem or object store through the CephNFS custom
+resource definition. This will spin up a cluster of
+[NFS Ganesha](https://github.com/nfs-ganesha/nfs-ganesha) servers that coordinate with one another
+via shared RADOS objects. The servers will be configured for NFSv4.1+ access only, as serving
```

Comment:
Here it is mention nfsv4.1+ but all the below examples is defined with nfsv4

Reply:
I think that's fine in the two other places where it appears. The .1+ isn't strictly necessary to know for those since v4.1 is also still v4.